huggingface/script/clm_requirements.txt

```diff
@@ -0,0 +1,6 @@
+accelerate
+torch >= 1.3
```

Comment:
where is this version come from? 1.3 seems very old version

Reply:
The requirements were taken from the requirements file from the transformers/examples repo

Reply:
I see, keep it as it is, we might consider update this as this seems very old